drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -446,6 +446,33 @@ def project_configs
       Hmis::ProjectConfig.all
     end
 
+    field :can_project_accept_referral, Boolean, null: false do
+      argument :project_id, ID, required: true
+      argument :referrer_enrollment_id, ID, required: true
+    end
+    def can_project_accept_referral(project_id:, referrer_enrollment_id:)
+      raise 'access denied' unless current_user.can_manage_outgoing_referrals?
```

Comment:
So this needs to ensure that the caller can refer out from the specific source project, not just that they can refer out from _any_ project. The tests should reflect this as well.

You can check it with: `current_permission?(permission: :can_manage_outgoing_referrals, entity: source_enrollment.project)`